packages/app-builder/src/components/Decisions/TriggerObjectDetail.tsx

```diff
@@ -27,13 +27,12 @@ export const TriggerObjectDetail = ({
         {t('decisions:trigger_object.type')}
       </Collapsible.Title>
       <Collapsible.Content>
-        <div className="flex flex-col gap-2">
+        <div className="grid grid-cols-[max-content_1fr] gap-2 break-all">
           {parsedTriggerObject.map(([property, data]) => (
-            <div key={property}>
-              <span className="font-semibold capitalize">{property}:</span>
-              &nbsp;
+            <React.Fragment key={property}>
```

Comment:
what is the difference in practice ?

Reply:
To clarify :
- `<div>` is a "real" tag element in the DOM
- `<React.Fragment/>` is a tag element in the virtual DOM but do not create a real tag element in the DOM 

Here, I need to move to a `grid` css layout. Grid do not require "rows" to be represented by intermediate `<div>` element (you directly pass an array of cells). So I need to remove the `<div />` to make it works (if you let divs, it will assume each div is a cell).
Since I do a map, I need to put keys on each element (React requirement), to this end, I use a Framgent.


NB: `<> ... </>` is a suggar notation for `<React.Fragment> ... </React.Fragment>` but you can't add attributes (=props) to the suggar notation